src/additional/mobileRU.js

```diff
@@ -0,0 +1,6 @@
+/*  mobile number for Russia Fedaration */
+$.validator.addMethod( "mobileRU", function( phone_number, element ) {
+	phone_number = phone_number.replace( /\(|\)|\s+|-/g, "" );
```

Comment:
Please create a new variable `phoneNumber` or something else and assign the result to it.